app/models/report.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 class Report < ApplicationRecord
-  LINK_MATCHER = %r{http://localhost:3000/reports/([1-9]+\d*)}
+  LINK_MATCHER = %r{http://localhost:3000/reports/([1-9]+[0-9]*)}
```

Comment:
> 自分の正規表現の引き出しから導き出しましたが[1-9]+[0-9]*の方が自然でしょうか？

なるほど、オリジナルだったんですね。個人的には`\d*`でもいいと思います。

```suggestion
  LINK_MATCHER = %r{http://localhost:3000/reports/(\d+)}
```

でも必要十分かな、とは思いますが、このへんは好みやこだわりの問題かもしれません

Reply:
`/reports/0`を含めたくないがためにこのような記述にしていたのですが、よく考えたら`mentioning_reports_in_content`で弾かれるので`\d*`で十分でした。